parsing_tools.py

```diff
@@ -36,26 +38,47 @@ def attachment_type(self):
         return DEFAULT_FILE_TYPE
 
 
-def parse_any_format(contents):
+def parse_any_format(contents: str) -> List[Message]:

```

Comment:
I don't know how typehints work, now it will typecheck during the run?

Reply:
Not during the run if I remember correctly, but before the run it will notice if you do something wrong (like try sending an int to this function, for example). We can also use a typechecking library like [mypy](https://mypy.readthedocs.io/en/stable/getting_started.html), though it will throw errors that aren't real because of some really unimportant issues that come with the Pandas library